src/app/about/about.scss

```diff
@@ -0,0 +1,12 @@
+
+.container {
+  display: flex;
```

Comment:
Since @alchuu00 started the convention.

I would like all scss files to have top level .body

Reply:
I will do that.